added New Linter Features And Pages

gisachris opened the pull request:
in this PR i added:
linter configurations
html index page
styling css page
descriptive readme file

Shedrack-Sunday commented on README.md:
```diff
@@ -0,0 +1,25 @@
+**[hello microverse]** is a web display of my first project in microverse
+
+## 🛠 Built With <a name="built-with"> html and css</a>
+
```
- [ ] You did well adding the README, however, you need to use the [Template](https://github.com/microverseinc/readme-template) provided with headers present. The `FAQ` section is the only optional section you can leave out. 

- [ ] All the other headers should be personalized based on the current project. Add all fields and as I said earlier, only the 'FAQ' is optional.

- [ ] I've attached a photo to show you all fields needed to be added. This field helps users know what your project entails, and give a professional look 👍 

## Headers
![image](https://user-images.githubusercontent.com/50990925/227226243-db392371-32c8-4ab8-9245-f21c47190e98.png)

Shedrack-Sunday reviewed (changes requested):
Hi @gisachris     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights:
✔️ good use of .gitignore
✔️ good PR summary
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- Some sections of your README  are not added. kindly check the review below to see the sections missing


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

gisachris committed:
updated README.md file

i have finished all the required changes from the previous code reviewer and i hope i have passed as this is my extra review of the week 
thank you.

souad988 commented on README.md:
```diff
@@ -0,0 +1,168 @@
+
+### Key Features <a name="key-features">key features</a>
+
+> Describe between 1-3 key features of the application.
+
```
Very well done so far , one small thing is that you need to remove default text under each section after adding your own content

souad988 commented on README.md:
```diff
@@ -0,0 +1,168 @@
+## 👥 Authors <a name="authors"> gisa chris</a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Kindly put your valid social account links here to let users connect with you , broken links always makes a repo look less professional

souad988 reviewed (changes requested):
Hi @gisachris , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔No linter errors
✔Good use of github workflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
update on README.md

DuaneDave reviewed (approved):
Hi @gisachris 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Correct usage of GitHub flow
✔️ The linters are correctly set up

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
added new linter features and pages
